### ISSUE TITLE
Some kotlin native stdlib types show up as ERROR CLASS

### ISSUE THREAD

**Issue by evant** [Kotlin/dokka#736]
**Describe the bug**
It appears that some stdlib types are showing up as `<ERROR CLASS>` in the documentation when including native targets. For example:

```kotlin
fun stdlibIssuePair(): Pair<String, String> = TODO()

fun stdlibIssueRegex(): Regex = TODO()
```

in `commonMain` produces the output:
```
stdlibIssuePair   (common, js, jvm) fun stdlibIssuePair(): Pair<String, String>
                  (linux) fun stdlibIssuePair(): <ERROR CLASS><String, String>
stdlibIssueRegex  (common, js, jvm) fun stdlibIssueRegex(): Regex
                  (linux) fun stdlibIssueRegex(): <ERROR CLASS>
```

**Expected behaviour**

I would expect only a single declaration without `<ERROR CLASS>`, as these types are available on native (see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-pair/ and https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-regex/).

**To Reproduce**
Sample repo: https://github.com/evant/dokka-issues

**Dokka configuration**

```groovy
dokka { 
    outputFormat = 'html'
    outputDirectory = "$buildDir/javadoc/"
    multiplatform {
        common {}
        js {}
        jvm {}
        linux {}
    }
}
```

**Installation**
- Operating system: Linux
- Build tool: Gradle v6.2
- Dokka version: 0.10.1

**Comment by kamilok1965** [comment]
Hi, thanks for reporting this bug!
Due to the fact that `klib` API is not yet stable, dokka is really dependent on the compiler version used to compile them. Could you try to downgrade your Kotlin compiler version to `1.3.61` ?

**Comment by evant** [comment]
Tried downgrading the sample project to `1.3.61`, seeing the same issue.

**Comment by kamilok1965** [comment]
Oh, I see the problem. Dokka can't recognize an unpacked `klib` such as this `stdlib` one.  It;ll be fixed in the next release. Meanwhile as a workaround you can probably generate the documentation using the `common` analyzer:
```
        linux {
            platform = "common"
        }
```

**Comment by msink** [comment]
Seems still not fixed in 1.4.0-rc

Tried refresh docs of [kotlin-libui](https://github.com/msink/kotlin-libui/tree/dokka) - too many &lt;ERROR CLASS&gt; errors.

To reproduce:  run `./gradlew -PpublishMode dokkaGfm`

**Comment by IgnatBeresnev** [comment]
It's gotten much better in the recent versions after a number of bugs have been fixed. Please update to the latest release and see if it's gone. For instance, Ktor used to have 1773 `ERROR CLASS` and after updating to 1.7.20 it's down to only 173.

If you're still seeing unresolved error classes, it's most likely that Dokka is unable to load and analyze some native code or `.klib` libraries. However, the exact problem differs from case to case, so the general recommendation is to look for the problem in project/dependency configuration.

If nothing helps and you're experiencing the issue, please create a separate issue with details on how to reproduce it so that we can figure it out. In the meantime, I'll close the current issue as outdated / too broad.
